### DISCUSSION TITLE
Turning Zehn into a live skin.

### DISCUSSION THREAD

**Discussion by yurisuika** [yurisuika/Zehn#2] [Announcements]
So, since Zehn is somewhat wrapping up in the bulk of the unfinished work, I have made my first attempt to make this into a live skin. This means that anyone should be able to download it once and probably should not need to download anything again, unless I add some new config options that break stuff.

Until I have settled on what course to take, the `master` branch is the regular skin and the `live` branch is what is needed for a purely live skin.

Will anyone read this? Who knows! Apparently Discussions feature is disabled by default.

**Comment by xiaohuaihai** [comment]
So, how to download the live one ? Just choice the live branch and still the same last release at the right of the website
